packages/drivers/odsp-driver/src/odspDocumentStorageServiceBase.ts

```diff
@@ -205,6 +212,12 @@ export abstract class OdspDocumentStorageServiceBase implements IDocumentStorage
 		const appTree = snapshotTree.trees[".app"];
 		const protocolTree = snapshotTree.trees[".protocol"];
 
+		if (isPartialSnapshot(snapshotTree)) {
+			return {
+				...snapshotTree,
+				...this.combineProtocolAndAppSnapshotTree(appTree, protocolTree),
```

Comment:
This looks fishy. Don't we include same information twice? I.e. tree will show up (in different forms) twice here.